sysbuild/Kconfig.matter

```diff
@@ -37,7 +42,11 @@ config MATTER_OTA_IMAGE_FILE_NAME
 	help
 	  Provides the file name of the generated Matter OTA image in the build directory.
 
+if SOC_SERIES_NRF54HX
+
+# Set the default value of manifest variant dedicated to Matter
 config SUIT_BASE_MANIFEST_VARIANT
 	default "matter/v1"
```

Comment:
this has the same issue I said before, if you configure an application without matter, then enable matter, this is not transferred and the original default will remain

Reply:
I wonder why users even may want to compile matter samples without matter. Honestly, it doesn't make sense to me. Moreover, I cannot set string values via `imply` or `select`, right? So it should be set as 

```
config SUIT_BASE_MANIFEST_VARIANT
     default "matter/v1" if MATTER
```
In the original definition within the SUIT module. 

Your comment is not related to the scope of this PR, and moreover, this default here wasn't written by us, but by the ncs-charon team some time ago and this change must be discussed with them as well. So could you please review the actual changes?